lib/jsonld-signatures.js

```diff
@@ -364,6 +357,32 @@ function _getSupportedAlgorithms() {
   return Object.keys(api.suites).filter(s => s !== 'LinkedDataSignature');
 }
 
+function _isSupportedProofPurpose(purpose) {
+  if(!purpose) {
+    // TODO: Enable error in the next major release
+    // throw new Error(`Please provide 'purpose' of the proof to verify.`);
+    return;
+  }
+  try {
+    api.proofPurposes.use(purpose);
```

Comment:
The `injector.use` API does not throw when an unknown handler is specified, it just returns undefined.

Reply:
It didn't return undefined when I used it. Here's a test I created that passes an unknown proof purpose https://github.com/digitalbazaar/jsonld-signatures/pull/39/commits/1614348db531fbe67bbe253df27cb5e117297c61

Reply:
Here's the injector, it does not throw: https://github.com/digitalbazaar/jsonld-signatures/blob/master/lib/Injector.js

Reply:
and if you are seeing an error here, what is it?

Reply:
The error is thrown after executing [Line 45](https://github.com/digitalbazaar/jsonld-signatures/blob/master/lib/Injector.js#L45). It's due to `require('UnknownProofPurpose')`, which also brings up another issue. If we passed in a proofPurpose that we didn't register AND it matches a name of a library installed in `node_modules/`, the current behavior of the Injector will require the already existing lib. This will definitely cause some unexpected errors.

Reply:
hmmm... yeah, we don't want any of that loading of default business for the proofPurpose.

maybe we need a kill switch option to the injector like `useDefaults: false` when using it in the proof purpose context.

Reply:
Should we name the parameter `useDefault` or `useDefaults`? I like the singular version.

Reply:
+1 to `useDefault` as I prepare to lose another singular vs plural argument.